badge.svg

```diff
@@ -0,0 +1,16 @@
+<svg width="100" height="20" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could we also create the badge on-the-fly instead of having the badge with the number git-versioned?